README.md

```diff
@@ -61,21 +62,59 @@ __Please be aware that although OpenMage is compatible that one or more extensio
 
 ## Installation
 
+### Manual Install
+
+Download the latest [release archive](https://github.com/OpenMage/magento-lts/releases) and extract it over your existing install. **Important:** you must download the ZIP file from a tagged version on the releases page, otherwise there will be missing dependencies.
+
 ### Composer
 
-Download the latest archive and extract it, clone the repo, or add a composer dependency to your existing project like so:
+Step 1: Create a new composer project:
 
 ```bash
-composer require "openmage/magento-lts":"^19.5.0"
+composer init
 ```
 
-To get the latest changes use:
+Step 2: Configure required install options. You can see all options [here](https://github.com/AydinHassan/magento-core-composer-installer#configuration).
 
 ```bash
-composer require "openmage/magento-lts":"dev-main"
+composer config --json extra.enable-patching true
+composer config extra.magento-core-package-type magento-source
+composer config extra.magento-root-dir htdocs
```

Comment:
do we have to add this line? at the end of the day composer asks for the root-dir when run

Reply:
I get an error without it:

```
$ composer require "openmage/magento-lts":"^20.0.0"
./composer.json has been updated
Running composer update openmage/magento-lts
Loading composer repositories with package information
Updating dependencies
Lock file operations: 16 installs, 0 updates, 0 removals
  - Locking eloquent/enumeration (5.1.1)
  - Locking flyingmana/composer-config-reader (20.0.1)
  - Locking justinrainbow/json-schema (5.2.12)
  - Locking magento-hackathon/magento-composer-installer (4.0.2)
  - Locking openmage/magento-lts (v20.0.18)
  - Locking psr/container (1.1.2)
  - Locking symfony/console (v5.4.19)
  - Locking symfony/deprecation-contracts (v2.5.2)
  - Locking symfony/polyfill-ctype (v1.27.0)
  - Locking symfony/polyfill-intl-grapheme (v1.27.0)
  - Locking symfony/polyfill-intl-normalizer (v1.27.0)
  - Locking symfony/polyfill-mbstring (v1.27.0)
  - Locking symfony/polyfill-php73 (v1.27.0)
  - Locking symfony/polyfill-php80 (v1.27.0)
  - Locking symfony/service-contracts (v2.5.2)
  - Locking symfony/string (v5.4.19)
Writing lock file
Installing dependencies from lock file (including require-dev)

In Options.php line 179:
                                                      
  magento-root-dir must be specified in root package  
                                                      

require [--dev] [--dry-run] [--prefer-source] [--prefer-dist] [--prefer-install PREFER-INSTALL] [--fixed] [--no-suggest] [--no-progress] [--no-update] [--no-install] [--no-audit] [--audit-format AUDIT-FORMAT] [--update-no-dev] [-w|--update-with-dependencies] [-W|--update-with-all-dependencies] [--with-dependencies] [--with-all-dependencies] [--ignore-platform-req IGNORE-PLATFORM-REQ] [--ignore-platform-reqs] [--prefer-stable] [--prefer-lowest] [--sort-packages] [-o|--optimize-autoloader] [-a|--classmap-authoritative] [--apcu-autoloader] [--apcu-autoloader-prefix APCU-AUTOLOADER-PREFIX] [--] [<packages>...]
```

Reply:
mmmm i get this

```
fab@MBP-Fab /t/prova> /usr/local/opt/php@7.3/bin/php /usr/local/bin/composer require "openmage/magento-lts":"^20"

./composer.json has been created
Running composer update openmage/magento-lts
Loading composer repositories with package information
Updating dependencies
Lock file operations: 16 installs, 0 updates, 0 removals
  - Locking eloquent/enumeration (5.1.1)
  - Locking flyingmana/composer-config-reader (20.0.1)
  - Locking justinrainbow/json-schema (5.2.12)
  - Locking magento-hackathon/magento-composer-installer (4.0.2)
  - Locking openmage/magento-lts (v20.0.18)
  - Locking psr/container (1.1.1)
  - Locking symfony/console (v5.4.19)
  - Locking symfony/deprecation-contracts (v2.5.2)
  - Locking symfony/polyfill-ctype (v1.27.0)
  - Locking symfony/polyfill-intl-grapheme (v1.27.0)
  - Locking symfony/polyfill-intl-normalizer (v1.27.0)
  - Locking symfony/polyfill-mbstring (v1.27.0)
  - Locking symfony/polyfill-php73 (v1.27.0)
  - Locking symfony/polyfill-php80 (v1.27.0)
  - Locking symfony/service-contracts (v2.5.2)
  - Locking symfony/string (v5.4.19)
Writing lock file
Installing dependencies from lock file (including require-dev)
Package operations: 16 installs, 0 updates, 0 removals
  - Installing justinrainbow/json-schema (5.2.12): Extracting archive
  - Installing eloquent/enumeration (5.1.1): Extracting archive
  - Installing flyingmana/composer-config-reader (20.0.1): Extracting archive
  - Installing symfony/polyfill-php80 (v1.27.0): Extracting archive
  - Installing symfony/polyfill-mbstring (v1.27.0): Extracting archive
  - Installing symfony/polyfill-intl-normalizer (v1.27.0): Extracting archive
  - Installing symfony/polyfill-intl-grapheme (v1.27.0): Extracting archive
  - Installing symfony/polyfill-ctype (v1.27.0): Extracting archive
  - Installing symfony/string (v5.4.19): Extracting archive
  - Installing symfony/deprecation-contracts (v2.5.2): Extracting archive
  - Installing psr/container (1.1.1): Extracting archive
  - Installing symfony/service-contracts (v2.5.2): Extracting archive
  - Installing symfony/polyfill-php73 (v1.27.0): Extracting archive
  - Installing symfony/console (v5.4.19): Extracting archive
magento-hackathon/magento-composer-installer contains a Composer plugin which is currently not in your allow-plugins config. See https://getcomposer.org/allow-plugins
Do you trust "magento-hackathon/magento-composer-installer" to execute code and wish to enable it now? (writes "allow-plugins" to composer.json) [y,n,d,?] y
  - Installing magento-hackathon/magento-composer-installer (4.0.2): Extracting archive
you may want to add the packages.firegento.com repository to composer.
add it with: composer.phar config -g repositories.firegento composer https://packages.firegento.com
  - Installing openmage/magento-lts (v20.0.18): Extracting archive
7 package suggestions were added by new dependencies, use `composer suggest` to see details.
Generating autoload files
11 packages you are using are looking for funding.
Use the `composer fund` command to find out more!
please define your magento root dir [root]

```

Reply:
but then I get the usual "./app/Mage.php is not a file." error.

the thing is I don't super like the "htdocs" folder :-\

Reply:
doesn't vendor come with an htaccess? anyway I see your point. couldn't we call it "pub" which is more like the modern projects do?

Reply:
> couldn't we call it "pub" which is more like the modern projects do?

I'll call it pub, and also put a comment there that they can call it whatever.

Reply:
I added a comment to each of those lines to help explain each one. Hopefully it's a bit better.